packages/lerna-semantic-release-analyze-commits/index.js

```diff
@@ -1,14 +1,23 @@
 var commitAnalyzer = require('@semantic-release/commit-analyzer');
 var log = require('lerna-semantic-release-utils').log;
+var affectsDelimiter = 'affects:';
+
+function findAffectsLine(commit) {
+  var message = (commit && commit.message) ? commit.message : '';
+  var affectsLine = message.split('\n\n').filter(function (line) {
```

Comment:
I guess that's a bit better, since it now checks for `affects:` that's probably safe